### PR TITLE
Take2:  Pat, can you OK these?

### PR DESCRIPTION
**Pull request by lukefahr**

### CODE REVIEW

File: m3/ice.py [hunk 1]
```diff
@@ -318,6 +321,7 @@ def spawn_handler(self, msg_type, event_id, length, msg):
                 logger.warn("Suppressed.")
             return
         try:
+
```
Comment (ppannuto) [inline m3/ice.py]: nit: random whitespace

File: m3/m3_common.py [hunk 2]
```diff
@@ -329,7 +329,8 @@ def __init__(self, args_list = None):
             self._parse_args(args_list)
             self.ice = ICE()
             self.callback_q = Queue.Queue()
-            self.install_handler()
+            #Messes up mbus_snooper, why was this here?
+            #self.install_handler()
```
Comment (ppannuto) [inline m3/m3_common.py]: yeah.. so, reading back, this was Pat trying to do too much with classes and inheritance, and I convinced myself that it was a good idea for there to be one "install handler" function (which, we'll note is one bloody line) that every "type" of "programmer" (goc / ein / mbus) would want to listen to their messages, so you have stuff like this:

```python
   class ein_programmer(object):
    TITLE = "EIN Programmer"
    DESCRIPTION = "Tool to program M3 chips using the EIN protocol."
    MSG_TYPE = 'b+'
```

Which will rely on the parent object to call install_handler during init but read the magic value of `MSG_TYPE` from `self` which will in this case change to `b+`....

A long-winded way of saying that `install_handler` was a bad idea and can/should be deleted

File: m3/m3_common.py [hunk 3]
```diff
@@ -940,10 +941,10 @@ def __init__(self, args, ice, callbacks=None):
...
         self.ice.msg_handler['b++'] = self._callback
 
-        self.ice.ice_set_baudrate_to_2000000()
-        def _atexit_reset_baudrate():
-            self.ice.ice_set_baudrate_to_115200()
-        atexit.register(_atexit_reset_baudrate)
+        #self.ice.ice_set_baudrate_to_2000000()
+        #def _atexit_reset_baudrate():
+        #    self.ice.ice_set_baudrate_to_115200()
+        #atexit.register(_atexit_reset_baudrate)
```
Comment (ppannuto) [inline m3/m3_common.py]: The new boards are faster, just commit and delete this

**Comment by lukefahr** [comment]
I did it with a different commit

**Pull request closed without merging**